src/stylesheets/menu.scss

```diff
@@ -0,0 +1,39 @@
+@import "colors";
+@import "buttons";
+
+.menu {
+  width: 8.75rem;
+}
+
+.menu-list {
+  background-color: $tc-white;
+  box-shadow: 0rem 0rem 0.5rem rgba(37, 40, 43, 0.12);
```

Comment:
Let's declare a color variable for this rgb. Maybe name it "shadow-gray" hhh

Reply:
rgba(37, 40, 43, 0.12) is quite similar to $light-gray, I will just reuse that instead of creating a new variable